tools/api-markdown-documenter/README.md

```diff
@@ -68,23 +68,34 @@ Are you already generating `.api.json` report files as a part of your build?
 
 If yes, create a file called `api-markdown-documenter.js` and paste the following code:
 
-```javascript
-const { loadModel, MarkdownRenderer } = require("@fluid-tools/api-markdown-documenter");
+<!-- AUTO-GENERATED-CONTENT:START (INCLUDE_CODE:path=./examples/RenderMarkdown.ts&language=typescript&start=5) -->
```

Comment:
Sample output replacing the current, hard-coded (and _incorrect_) sample usage code with content directly embedded from a "live" code example that is build-validated.